Build/linux/build.sh

```diff
@@ -83,13 +73,7 @@ else
 	AR_COMPILER=`dirname $ICC_COMPILER`/xiar
 fi
 
-if [ "$1" == "cpp" ]; then
-	COMPILE_AS_CPP=1
-elif [ "$2" == "cpp" ]; then
-	COMPILE_AS_CPP=1
-else
-	COMPILE_AS_CPP=0
-fi	
+
```

Comment:
Can we keep the code to build as cpp? In case others want to add some cpp code to sample app.